Userland/Libraries/LibJS/Runtime/VM.h

```diff
@@ -114,8 +114,16 @@ class VM : public RefCounted<VM> {
     // https://tc39.es/ecma262/#running-execution-context
     // At any point in time, there is at most one execution context per agent that is actually executing code.
     // This is known as the agent's running execution context.
-    ExecutionContext& running_execution_context() { return *m_execution_context_stack.last(); }
-    ExecutionContext const& running_execution_context() const { return *m_execution_context_stack.last(); }
+    ExecutionContext& running_execution_context()
+    {
+        VERIFY(!m_execution_context_stack.is_empty());
```

Comment:
Big fan of this change, and the following one! Just wondering though, just given how often this happens - for people who aren't familiar with the issue, I wonder if it's also worth us leaving a comment / mention the workaround that we often need to make for this in LibWeb-land

Reply:
I don't think this needs a comment for the workaround here in LibJS. It feels like bad layering, and we are familiar enough with the issue that a quick "hey what's this execution context stack thing about" in #browser will yield the right answer.